src/logger.rs

```diff
@@ -86,6 +89,12 @@ impl Logger {
             StdoutLog::Off => {}
         }
 
+        if !config.log_gfx_device {
+            logger.dispatch = logger
+                .dispatch
+                .filter(|m| !m.target().starts_with("gfx_device_gl"));
```

Comment:
This is quite nuclear, couldn't you do perhaps
```rust
pub gfx_device_level: Option<LevelFilter>,
```
in the struct with a default of `Warn` and then
```rust
if let Some(level) = config.gfx_device_level {
    logger.dispatch = logger.dispatch.level_for("gfx_device_gl", level);
}
```
here?

Reply:
That's way better, thanks!